spec/directions/bake_index/v1_spec.rb

```diff
@@ -174,6 +174,24 @@
     expect(a_section.items.first.term_text).to eq 'temperature'
   end
 
+  it 'groups by number when word starts with a number' do
+    a_section.add_term(text_only_term('70S ribosome'))
+    a_section.add_term(text_only_term('3-PGA'))
+    a_section.add_term(text_only_term('5’ cap'))
+    a_section.add_term(text_only_term('3-phosphoglycerate'))
+
+    expect(a_section.items.map(&:term_text)).to eq ['3-PGA', '3-phosphoglycerate', '5’ cap', '70S ribosome']
```

Comment:
This is testing the sort, not the grouping.  Ditto below.  You might need to add number and accent mark terms into the larger bake spec to see that they go where they should